support.sh

```diff
@@ -4,34 +4,71 @@
 DOCKER="${DOCKER:-docker}"
 NSENTER="${NSENTER:-nsenter}"
 BRIDGE="${BRIDGE:-bridge}"
-BRCTL="${BRCTL:-brctl}"
 IPTABLES="${IPTABLES:-iptables}"
+IPVSADM="${IPVSADM:-ipvsadm}"
+IP="${IP:-ip}"
 
 NSDIR=/var/run/docker/netns
-BRIDGEIF=br0
 
 function die {
     echo $*
     exit 1
 }
 
-type -P ${DOCKER} > /dev/null || die "This tool requires the docker binary"
-type -P ${NSENTER} > /dev/null || die "This tool requires nsenter"
-type -P ${BRIDGE} > /dev/null || die "This tool requires bridge"
-type -P ${BRCTL} > /dev/null || die "This tool requires brctl"
-type -P ${IPTABLES} > /dev/null || die "This tool requires iptables"
+function echo_and_run {
+  echo "#" "$@"
+  eval $(printf '%q ' "$@") < /dev/stdout 2>/dev/null
+}
+
+type -P ${DOCKER} > /dev/null || echo "This tool requires the docker binary"
+type -P ${NSENTER} > /dev/null || echo "This tool requires nsenter"
+type -P ${BRIDGE} > /dev/null || echo "This tool requires bridge"
+type -P ${IPTABLES} > /dev/null || echo "This tool requires iptables"
+type -P ${IPVSADM} > /dev/null || echo "This tool requires ipvsadm"
+type -P ${IP} > /dev/null || echo "This tool requires ip"
+
+echo "Host Configuration"
+echo_and_run ${IPTABLES} -w1 -n -v -L -t filter | grep -v '^$'
+echo_and_run ${IPTABLES} -w1 -n -v -L -t nat | grep -v '^$'
+echo_and_run ${IPTABLES} -w1 -n -v -L -t mangle | grep -v '^$'
+printf "\n"
 
-echo "iptables configuration"
-${IPTABLES} -w1 -n -v -L -t filter
-${IPTABLES} -w1 -n -v -L -t nat
-echo ""
+echo "Host addresses and routes"
+echo_and_run ${IP} -o -4 address show
+echo_and_run ${IP} -4 route show
+printf "\n"
 
 echo "Overlay network configuration"
-for networkID in $(${DOCKER} network ls --filter driver=overlay -q) ; do
-    echo "Network ${networkID}"
-    nspath=(${NSDIR}/*-$(echo ${networkID}| cut -c1-10))
-    ${DOCKER} network inspect -v ${networkID}
-    ${NSENTER} --net=${nspath[0]} ${BRIDGE} fdb show ${BRIDGEIF}
-    ${NSENTER} --net=${nspath[0]} ${BRCTL} showmacs ${BRIDGEIF}
-    echo ""
+for networkID in $(${DOCKER} network ls --filter driver=overlay -q) "ingress_sbox"; do
```

Comment:
please switch to using long ids.  tested this on 17.06.2-ee-6.  seems to do the trick.
```
diff --git a/support.sh b/support.sh
index c948c043..857dfb4a 100755
--- a/support.sh
+++ b/support.sh
@@ -39,7 +39,7 @@ echo_and_run ${IP} -4 route show
 printf "\n"

 echo "Overlay network configuration"
-for networkID in $(${DOCKER} network ls --filter driver=overlay -q) "ingress_sbox"; do
+for networkID in $(${DOCKER} network ls --filter driver=overlay -q --no-trunc) "ingress_sbox"; do
     echo "nnn Network ${networkID}"
     if [ "${networkID}" != "ingress_sbox" ]; then
         nspath=(${NSDIR}/*-${networkID:0:10})
```